.github/workflows/_mac-build.yml

```diff
@@ -71,7 +71,7 @@ jobs:
 
       # [see note: pytorch repo ref]
       - name: Checkout PyTorch
-        uses: pytorch/pytorch/.github/actions/checkout-pytorch@main
+        uses: pytorch/pytorch/.github/actions/checkout-pytorch@malfet-patch-1
```

Comment:
This is temporary for testing I guess?

Reply:
Yes, because otherwise it's a no-op that could not be tested...